src/components/Pf/PfTitle.tsx

```diff
@@ -68,15 +79,30 @@ class PfTitle extends React.Component<PfTitleProps, PfTitleState> {
         break;
       default:
     }
-
-    this.state = {
+    this.setState({
       namespace: ns,
       type: type,
       name: name,
       graphType: graphType,
       icon: icon,
       cytoscapeGraph: cytoscapeGraph.build()
-    };
+    });
+  }
+
+  componentDidMount(): void {
```

Comment:
I don't think we historically have added `void` type annotations for functions, but that's just a nitpick.

Reply:
We have them in other locations.

Reply:
Np then :)